### ISSUE TITLE
Product CSV bools

### ISSUE THREAD

**Issue by jtrotsky** [ShoppinPal/vend-tools#8]
In a Vend product CSV, there are some strict rules for what is a valid import to a Vend store. 
The two boolean columns in the CSV are `active` and `track_inventory`. 

Vend will only accept a 1 or 0 for bools, true and false are not valid. (this is our error.. but it exists either way.)

Currently vend-tools downloads products with `active` and `track_inventory` like this:

![screenshot 2015-03-04 09 48 27](https://cloud.githubusercontent.com/assets/5625795/6485745/cec5b082-c253-11e4-9401-7ab7cdbe2ec1.png)

It needs to looks like this:

![screenshot 2015-03-04 09 48 51](https://cloud.githubusercontent.com/assets/5625795/6485746/cec79e4c-c253-11e4-8103-9a33fb3daa75.png)

Should be a simple fix. 
I don't know how it should look here, but this is what I did (in Golang):

```
// Both must be integer bools instead of the default strings.
a := 0
if *product.Active {
    a = 1
}
active := fmt.Sprintf("%v", a)

t := 0
if *product.TrackInventory {
    t = 1
}
trackInventory := fmt.Sprintf("%v", t)
```

**Comment by pulkitsinghal** [comment]
@jtrotsky - Thanks! Yes, this should be accomplishable.

@elbrando - You use this feature quite a bit to print new barcodes right? By piping the data into your 3rd party software every so often ... correct? So, I want to make sure that if I make the changes suggested above ... replace true/false with 1/0 ... it should not break your process ... will it? Let us know.

**Comment by pulkitsinghal** [comment]
I'm thinking of an explicit flag to enable the functionality, something like:
1. --formatForVend true, or
2. --vendCompatible true, or
3. --vendFriendly true, or
4. --vendFormatted true, or
5. --dataFormat vend

**Comment by elbrando** [comment]
Yes, we use this a lot. The "true/false" vs. "0/1" thing is what it is. Should the field be true/false? Sure. Can we deal with the bifurcation? Sure.

Personally, I'd rather deal with the 0/1 until Vend makes it evaluate true/false, as it should. ;)

Mostly, I just wanted to use "bifurcation" in a meaningful way today. Thanks.

**Comment by pulkitsinghal** [comment]
Soooo ... (god i'm dumb)... @elbrando you want `0/1` convention right?

**Comment by jtrotsky** [comment]
@pulkitsinghal I like the idea of a --vendFriendly flag.

> Personally, I'd rather deal with the 0/1 until Vend makes it evaluate true/false, as it should. ;)

Vend currently only evaluates `1/0`, but yes it it definitely _should_ take true/false also. :)
